src/webui/www/private/index.html

```diff
@@ -2,6 +2,14 @@
 <html lang="${LANG}">
 
 <head>
+    <script>
+        let path = window.location.pathname;
+        if (path) {
+            let lastChar = path.substr(path.length - 1);
```

Comment:
`substr` is not recommended: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr#browser_compatibility
I would suggest: `const lastChar = path.slice(-1);`